app/src/main/java/dev/andrylat/raqimbek/bankingutils/CardValidationInfo.java

```diff
@@ -0,0 +1,21 @@
+package dev.andrylat.raqimbek.bankingutils;
+
+import java.util.List;
+
+public class CardValidationInfo {
+    boolean isValid;
+    List<String> errors;
+    
+    CardValidationInfo(boolean isValid, List<String> errors) {
```

Comment:
Are you sure that your contructor should have default access? It means that it can't be used outside of current package. Is it valid option?